sharegpt/sharegpt-qa.py

```diff
@@ -82,23 +83,50 @@ async def _async_request(self, messages, max_tokens: int) -> Response:
         body = ""
 
         try:
-            stream = await self.client.chat.completions.create(
-                messages=messages,
-                model=self.model,
-                temperature=0,
-                stream=True,
-                max_tokens=max_tokens,
-                stream_options={"include_usage": True},
-            )
-
-            async for chunk in stream:
-                if not chunk.choices:
-                    continue
-                delta = chunk.choices[0].delta.content
-                if delta:
-                    if first_token is None:
-                        first_token = time.time()
-                    body += delta
+            # Check if we should use chat completions API
```

Comment:
Is there a motivation for using both ``completions`` and ``chat.completions`` API? It is definitely okay to have both, it's just I am not aware of the individual use cases.

Reply:
Hi, @Siddhant-Ray , yes, we have a project that needs to test both APIs, and the project right now is using LMBenchmarking for testing.